packages/sdk/src/lend/providers/morpho/lend.supersim.test.ts

```diff
@@ -0,0 +1,278 @@
+import { createWalletClient, erc20Abi, http, parseUnits } from 'viem'
+import { unichain } from 'viem/chains'
+import { afterAll, beforeAll, describe, expect, it } from 'vitest'
+
+import type { VerbsInterface } from '../../../types/verbs.js'
+import { setupSupersimTest, stopSupersim } from '../../../utils/test.js'
+import { initVerbs } from '../../../verbs.js'
+import { SUPPORTED_VAULTS } from './vaults.js'
+
+// Use the first supported vault (Gauntlet USDC)
+const TEST_VAULT = SUPPORTED_VAULTS[0]
+const USDC_ADDRESS = TEST_VAULT.asset.address
+const TEST_VAULT_ADDRESS = TEST_VAULT.address
+
+describe('Morpho Lend', () => {
+  let supersimProcess: any
+  let publicClient: any
+  let testAccount: any
+  let walletClient: any
+  let verbs: VerbsInterface
+  let testWallet: any
+
+  beforeAll(async () => {
+    // Set up supersim with funded wallet using helper
+    const setup = await setupSupersimTest({
+      supersim: {
+        chains: ['unichain'],
+        l1Port: 8546,
+        l2StartingPort: 9546,
+      },
+      wallet: {
+        rpcUrl: 'http://127.0.0.1:9546',
+        chain: unichain,
+        amount: '10',
+      },
+    })
+
+    supersimProcess = setup.supersimProcess
+    publicClient = setup.publicClient
+    testAccount = setup.testAccount
+
+    // Create wallet client for signing transactions
+    walletClient = createWalletClient({
+      account: testAccount,
+      chain: unichain,
+      transport: http('http://127.0.0.1:9546'),
+    }) as any
+
+    // Initialize Verbs SDK with Morpho lending
+    verbs = initVerbs({
+      wallet: {
+        type: 'privy',
+        appId: 'test-app-id',
+        appSecret: 'test-app-secret',
+      },
+      lend: {
+        type: 'morpho',
+        defaultSlippage: 50,
+      },
+      chainId: unichain.id,
+      rpcUrl: 'http://127.0.0.1:9546',
```

Comment:
should we remove these as arguments to verbs and replace it with a `chainsConfig?` that can plug into where we create the `ChainManager`: https://github.com/ethereum-optimism/verbs/blob/ca59c1b16e30c01c0166571c73bc888daef6ea16/packages/sdk/src/verbs.ts#L30?